libbpf-cargo/src/test.rs

```diff
@@ -771,10 +771,16 @@ fn test_skeleton_builder_basic() {
         #include "vmlinux.h"
         #include <bpf/bpf_helpers.h>
 
+        struct unique_key {{
+            int cap;
+            u32 tgid;
+            u64 cgroupid;
+        }};
+
         struct {{
                 __uint(type, BPF_MAP_TYPE_HASH);
                 __uint(max_entries, 1024);
-                __type(key, u32);
+                __type(key, struct unique_key);
```

Comment:
as I mentioned above, it's legal to have `__uint(key_size, 8);`, maybe let's add another test for this to make sure this is handled gracefully?